### ISSUE TITLE
Add taxon constraints to FAO

### ISSUE THREAD

**Issue by cmungall** [obophenotype/fungal-anatomy-ontology#13]
See also https://github.com/geneontology/go-ontology/issues/11313#issuecomment-151553984

**Comment by dianeoinglis** [comment]
Is "only_in_fungi" the type of constraint you want to add?
How granular do you think these should be? 
Some structures are not present in Ascomycetes, such as fused clamps which are structures made by Basidiomycetes, etc.

@cmungall 
cc @thomaspc

**Comment by cmungall** [comment]
For now I think high level distictions are best, at the Ascomycetes vs Basidiomycetes level seems good.

The main thing to be done is to do the initial plumbing so that you can add these. If you're working i oboedit this will take some futzing I'm afraid. Maybe we could explore keeping them in a separate file for now

**Comment by ValWood** [comment]
Yes  it would be good to have "never in Ascomycetes" where possible so we can hide them from our community curators.
Cheers

Val

**Comment by dianeoinglis** [comment]
@cmungall
Currently, the FAO.obo does not have taxon constraints. Do you want the constraints in the FAO or in the xref'd GO term or both?

**Comment by cmungall** [comment]
Ideally in the FAO, but this will take some work setting up as it involves imports. It's much easiest if all work is done in Protege thereafter. If that isn't a problem, I can make a pull request with the changes. (the source file can stay as .obo)

**Comment by dianeoinglis** [comment]
@cmungall
Could the goal of limiting the terms relevant to S. cerevisiae, etc be done as marked subsets as they are in the APO? For example, 'virulence' is marked as a Candida, Aspergillus and Cryptococcus subset so the cerevisiae curators never see the term, they only see terms in the S. cerevisiae subset. This seems cleaner and the majority of terms will not be in the Sc subset. I can't do the edits myself right now but I can work the specs out in the tracker. Right now there are about a dozen NTRs or modifications to make in addition to taxon limiters

**Comment by dianeoinglis** [comment]
@cmungall
I don't know why but the master file in GitHub is missing the [Typedef] is_a whereas it is present in the same file in OBO Foundry

**Comment by cmungall** [comment]
is_a shouldn't be declared, it's built in
